src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/SinkIslandCheck.java

```diff
@@ -32,21 +31,21 @@
  * @author cuthbertm
  * @author gpogulsky
  * @author savannahostrowski
+ * @author nachtm
  */
 public class SinkIslandCheck extends BaseCheck<Long>
 {
     private static final float LOAD_FACTOR = 0.8f;
     private static final long TREE_SIZE_DEFAULT = 50;
     private static final List<String> FALLBACK_INSTRUCTIONS = Collections
             .singletonList("Road is impossible to get out of.");
-    private static final String MOTORCYCLE_PARKING_AMENITY = "MOTORCYCLE_PARKING";
-    private static final String PARKING_ENTRANCE_AMENITY = "PARKING_ENTRANCE";
-    private static final List<String> amenityValuesToExclude = Arrays.asList(
-            AmenityTag.PARKING.toString(), AmenityTag.PARKING_SPACE.toString(),
-            MOTORCYCLE_PARKING_AMENITY, PARKING_ENTRANCE_AMENITY);
+    private static final AmenityTag[] amenityValuesToExclude = { AmenityTag.PARKING,
```

Comment:
nit: `amenityValuesToExclude` list name could be capitalized as other constants.